### ISSUE TITLE
Agenda for June 23 meeting

### ISSUE THREAD

**Issue by foolip** [web-platform-tests/interop-2022#90]
Here's the agenda for our meeting tomorrow:

- https://github.com/web-platform-tests/interop-2022/issues/78 proposal:
  - RFC for the process only, using [timeline from comment](https://github.com/web-platform-tests/interop-2022/issues/78#issuecomment-1151357104)
  - Rest of process run without RFC process
- https://github.com/web-platform-tests/interop-2022/issues/89
- Posting an update on how Interop 2022 has enabled developers (email from @robnyman)
- Test change review:
  - https://github.com/web-platform-tests/interop-2022/issues/81 ping
  - https://github.com/web-platform-tests/interop-2022/issues/88
- Investigation effort updates
  - Editing (@jgraham): https://github.com/web-platform-tests/interop-2022-editing
  - Pointer and Mouse Events (@flackr): https://github.com/web-platform-tests/interop-2022-pointer
  - Viewport (@bramus): https://github.com/web-platform-tests/interop-2022-viewport

Previous meeting: https://github.com/web-platform-tests/interop-2022/issues/87

**Comment by foolip** [comment]
_Rough notes from the meeting_

**Interop 2023**

Schedule in RFC will be https://github.com/web-platform-tests/interop-2022/issues/78#issuecomment-1151357104 + https://github.com/web-platform-tests/interop-2022/issues/78#issuecomment-1152159713

@jgraham: test review can run in parallel, from proposals time, but can keep tweaking the test list until shortly before launch when we take screenshots and such. But I hope that when we've selected proposals we trust in the test coverage we have.

@jensimmons: Does that amount to something like https://github.com/web-platform-tests/interop-2022/issues/78#issuecomment-1164606116?

@jgraham: Yes.

@jgraham: Generally for 2022 the tests were there, but in a few cases we wrote the tests afterwards.

[**Small features as a focus area for Interop 2023?**](https://github.com/web-platform-tests/interop-2022/issues/89)

@jgraham: This feels like scope creep and potentially high risk. To also include writing the spec text and going from zero spec+tests in one year, I think that's a pretty hard sell. Where there's real developer demand, it could come under and "investigate" area. But I think it's pretty hard to agree to implementing something in December that has zero spec text in January.

@jensimmons: (1) a bucket of small papercuts that web developers would be a fine proposal I think but (2) this is not the group to be writing specifications, this isn't the WHATWG or CSSWG.

@jgraham: +1 to it being totally reasonable to have a bucket for small features that are already specced but not widely implemented.

@bkardell: These are not all equal in terms of spec status and work. But I do agree there are things that we could get done, including the spec'ing, where it's a 6-8 month time period from spec'ing to implementing. Collecting ideas now is a good idea. We can look at this list and if someone has taken up the work and advanced it it could be a good candidate, but not if no work has happened. With an initial PR and some buy-in, that's a maybe.

@zcorpan: Seems like what @domenic would like is to align browser roadmaps on these small features that have web developer demand. That could be in scope for https://github.com/web-platform-dx which is about figuring out what web developer wants. The other part is getting it shipped in a somewhat synchronized manner. Maybe it doesn't have to be yearly since browsers ship more often.

@bkardell: I love bringing things like this to the group so we can help bring and align conversations to see if we can align any of these tho

@jensimmons: The purpose of this group should be clear. We could put pressure on groups to finish the spec so that we could include it. But I don't think it's our role to align roadmaps or to force us to implement something. But not writing specs. We can bring issues to the CSSWG for example, recent good example on color work there. But not coordinating work.

**Posting an update on how Interop 2022 has enabled developers**

@robnyman: Features we can mention? Cascade Layers, `<dialog>`, others? Interest in coordinating posting about those in the coming month, say?

@jensimmons: I think a mid-year update would be a good idea, a big chunk of that could be the specific things we've improved. Maybe we can draft and coordinate?

@robman: Everyone feel free to reply to the email thread too.

**Investigation effort updates**

Since we ran out of time for the meeting, @foolip will get a summary of each investigation effort and post an update on this issue + email everyone with a summary.

**Comment by foolip** [comment]
Investigation effort updates via chat/email that happened after the last meeting.

On pointer events, @flackr says:

> For the pointer interop meeting, we have had our initial meeting and agreed on the scope. We have a list of issues, though it may still grow in size. Perhaps it would be helpful for me to describe the decisions from our [first meeting](https://github.com/web-platform-tests/interop-2022-pointer/issues/3#issuecomment-1150223304) on the README. The scoring is likely to be what you suggested - a linear score of the issue list.
>
> I believe we have the right contacts for each browser engine, however I will likely need to set up followup meetings and assign responsible people to issues to ensure that we continue to make progress.
>
> TLDR; we have progress on the first stage which is harder to measure (unknown length of issues list) but I believe with a few gentle pushes we can accelerate progress on triaging the issues and getting tests together.

On viewport, @bramus says:

> Have been discussing with our engineering team on how our adjustments to the ICB would achieve interop, yet would also require an adjustment by authors should they want the old behavior back.

https://github.com/web-platform-tests/interop-2022-viewport/issues/3 tracks the next meeting.
